environment/waf.tf

```diff
@@ -0,0 +1,16 @@
+data "aws_wafv2_web_acl" "main" {
+  name  = "${local.account.name}-web-acl"
+  scope = "REGIONAL"
+}
+
+resource "aws_wafv2_web_acl_association" "admin" {
+  count        = local.account.associate_alb_with_waf_web_acl_enabled ? 1 : 0
+  resource_arn = aws_lb.front.arn
```

Comment:
you've got the front alb twice

Reply:
Ahh yep, can't believe I missed that. Cheers Andrew